src/mpid/ch4/netmod/ofi/ofi_coll_select_container_parsers.c

```diff
@@ -0,0 +1,108 @@
+
+/* -*- Mode: C; c-basic-offset:4 ; indent-tabs-mode:nil ; -*- */
+/*
+ *  (C) 2006 by Argonne National Laboratory.
+ *      See COPYRIGHT in top-level directory.
+ *
+ *  Portions of this code were written by Intel Corporation.
+ *  Copyright (C) 2011-2016 Intel Corporation.  Intel provides this material
+ *  to Argonne National Laboratory subject to Software Grant and Corporate
+ *  Contributor License Agreement dated February 8, 2012.
+ */
+
+#include <mpidimpl.h>
+#include "ofi_noinline.h"
+
+int MPIDU_SELECTION_MAX_OFI_ALGORITHM_INDICES[MPIDU_SELECTION_COLLECTIVES_MAX]
+    = {
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+};
+
+const char **MPIDU_SELECTION_OFI_ALGORITHMS[] = {
+    NULL,
```

Comment:
We need to find a way to initialize this with `= { NULL }` instead of each entry.

Reply:
Okay, will do that.